exclude_files.txt

```diff
@@ -0,0 +1,7 @@
+.git*
```

Comment:
Can we remove this file?

Reply:
The other temporary files [rest.http,.github/workflows, and.gitignore] are also appended to the .tgz file if we remove this. I decided not to remove this file.